test/stripe/bitcoin_transaction_test.rb

```diff
@@ -0,0 +1,21 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+module Stripe
+  class BitcoinTransactionTest < Test::Unit::TestCase
+    should "retrieve should retrieve bitcoin receiver" do
+      @mock.expects(:get).once.returns(make_response(make_bitcoin_transaction))
```

Comment:
Can we add a `with("#{Stripe.api_base}/v1/bitcoin/transactions/bttxn_test_transaction", nil, nil)` to the expects call to ensure that the right URL is being hit?